finger-exercises/ecmascript/01 - spread/test.js

```diff
@@ -0,0 +1,69 @@
+import { min, copy, reverseMerge, filterAttribs } from '.';
+
+describe('Spread operator', () => {
+  it('min returns the param if is the only argument', () => {
```

Comment:
Just as a suggestion, to keep scope, enable tests one by one.
You may implement a solution to a test and make many pass without knowing why they passed.